WcaOnRails/app/models/registration.rb

```diff
@@ -208,6 +208,12 @@ def self.wcif_json_schema
     }
   end
 
+  class << self
```

Comment:
Why is this `class << self` extension necessary here?

Reply:
hi @gregorbg I was just following the rubocop convention - (**_Use `class << self` to define a class method. (convention:Style/ClassMethodsDefinitions)_**). 

`def self.accepted_and_paid_pending_count` will work too. But this will throw a `rubocop` warning. I can use this convention if the `rubocop` suggested convention seems confusing. Should I make the change?

Reply:
Changed to old syntax in https://github.com/thewca/worldcubeassociation.org/pull/7458/commits/bac8dd0831b07a2018841e40ad1e57263e75bd24